modules/markup/markdown/markdown.go

```diff
@@ -184,18 +185,26 @@ func actualRender(body []byte, urlPrefix string, metas map[string]string, wikiMa
 			_ = lw.CloseWithError(fmt.Errorf("%v", err))
 		}()
 
-		pc := NewGiteaParseContext(urlPrefix, metas, wikiMarkdown)
-		if err := converter.Convert(giteautil.NormalizeEOL(body), lw, parser.WithContext(pc)); err != nil {
+		// FIXME: Don't read all to memory, but goldmark doesn't support
```

Comment:
exist an upstream issue for this?

Reply:
I think yes, I cannot found any function to allow `io.Reader`

Reply:
https://github.com/yuin/goldmark/discussions/209